.rubocop.yml

```diff
@@ -18,3 +18,7 @@ MethodLength:
 
 Style/FileName:
   Enabled: false
+
+Style/FormatString:
```

Comment:
I'd prefer if this was put on each line instead of for the whole file. I havent consolidated the `.rubocop.yml` it's on the todo at some point.